packages/store-api/src/platforms/vtex/loaders/simulation.ts

```diff
@@ -0,0 +1,42 @@
+import DataLoader from 'dataloader'
+
+import type {
+  PayloadItem,
+  Simulation,
+} from '../clients/commerce/types/Simulation'
+import type { Options } from '..'
+import type { Clients } from '../clients'
+
+export const getSimulationLoader = (_: Options, clients: Clients) =>
+  new DataLoader<PayloadItem[], Simulation>(
+    async (items) => {
+      const simulated = await clients.commerce.checkout.simulation({
+        items: [...items.flat()],
```

Comment:
This request to the simulation API doesn't respect the regionalization feature. You should add 
```ts
   items: [...items.flat()],
   shippingData: {
     logisiticInfo: [{ itemIndex: index of item on items list, regionId: "id" }],
   }

Reply:
Of course, no problem.